tutornotes/patches/openedx-lms-common-settings

```diff
@@ -1,3 +1,3 @@
 # Student notes
 EDXNOTES_CLIENT_NAME = "notes"
-EDXNOTES_INTERNAL_API = "http://notes:8000/api/v1"
+EDXNOTES_INTERNAL_API = "http://notes:8120/api/v1"
```

Comment:
This setting looks like it is shared between production and development. However, if I am reading the other configurations correctly, production does use port 8000 while only development uses 8120. If that is true, then modifying in place like this would fix development but at the price of breaking production. Instead, we might need to move this setting into the other files called `openedx-lms-development-settings` and `openedx-lms-production-settings` so the two environments can have distinct values.

Reply:
Once you make that change. Please first test that development works. Then please work with @edazzocaisser to deploy the branch to our production environment and test that it works before opening a pr to the parent repo.